### PR TITLE
Vertex Smearing, Tracker Digitization Information, BField Adjustment

### PR DESCRIPTION
**Pull request by vvolkl**
This adds a few missing pieces for the next productions. A FCCSW installation in `inits/Pre9_0_1.py`  can be used to test this configuration. (It should be properly tagged and released next week)

### CODE REVIEW

**Comment by vvolkl** [comment]
In the first Iteration I forgot to actually use the `smeartool` in the job. Now there are command line arguments for both pythia and gaudi vertex smearing.
The output should be comparable: I ran just the generation part of the minbias jobs and stored the result here:
```
/afs/cern.ch/work/v/vavolkl/public/output__vavolkl_201803081347161728_gauditoolVertexsmearing.root
/afs/cern.ch/work/v/vavolkl/public/output__vavolkl_201803081411031168_pythiaVertexsmearing.root
```

**Comment by vvolkl** [comment]
* Vertex time distribution with gauditool (Note that edm saves ctau, and 1 ns corresponds to 300 mm ctau, so this comes out correctly - with a smaller cut I get a std dev of 55 mm ~ 180 ps
genvertices ctau gauditoolvertexsmearing
* Vertex time distribution with pythiasmearing
genvertices ctau pythiavertexsmearing
* Vertex z distribution with gauditool
genvertices z gauditoolvertexsmearing
* Vertex z distribution with pythiasmearing
genvertices z pythiavertexsmearing

**Comment by vvolkl** [comment]
On a first glance, the digitisation information seems to make sense (note that there are two types of modules with different thicknesses, hence the two steps)

poststeppositioncheckx

File: config/geantSim.py [hunk 1]
```diff
@@ -68,10 +69,12 @@
 
 
 from Gaudi.Configuration import *
+from GaudiKernel import SystemOfUnits as units
 ##############################################################################################################
 #######                                         GEOMETRY                                         #############
 ##############################################################################################################
-path_to_detector = '/afs/cern.ch/work/h/helsens/public/FCCsoft/FCCSW-0.8.3/'
+#path_to_detector = '/afs/cern.ch/work/h/helsens/public/FCCsoft/FCCSW-0.8.3/'
+path_to_detector = "/afs/cern.ch/work/v/vavolkl/public/fcc.cern.ch/sw/pre0.9.1/FCCSW/InstallArea/"
```
Comment (clementhelsens) [inline config/geantSim.py]: can you change that to the official installed released? It pointed to my local due to the non installation of the tail catcher geo

**Pull request merged**